build/Makefile.in

```diff
@@ -475,6 +475,7 @@ UV_UNIX = 3rdparty/libuv/src/strscpy@obj@ \
           3rdparty/libuv/src/unix/loop@obj@ \
           3rdparty/libuv/src/unix/pipe@obj@ \
           3rdparty/libuv/src/unix/poll@obj@ \
+          3rdparty/libuv/src/unix/epoll@obj@ \
```

Comment:
I believe epoll is linux only, so this line should probably move to the `UV_LINUX` section.